proto/src/lib.rs

```diff
@@ -33,177 +33,224 @@ pub use state::StateReadProto;
 #[cfg(feature = "penumbra-storage")]
 pub use state::StateWriteProto;
 
-/// Core protocol structures.
-pub mod core {
-    /// Crypto structures.
-    pub mod crypto {
-        pub mod v1alpha1 {
-            include!("gen/penumbra.core.crypto.v1alpha1.rs");
+pub use penumbra::*;
+
+pub mod penumbra {
```

Comment:
Not sure how much we get out of putting all the penumbra code in a submodule that we re-export, but i guess it's not the worst to be consistent with the other vendored protos.